### PR TITLE
Dropdown menu background fix

### LINKED ISSUE #72: Navbar menu not visible with COPS logo
Screenshot 2020-02-05 at 15 45 49
The dropdown menu and COPS logo are getting intermixed. The dropdown menu should have a background so that its contents doesn't get mixed with the COPS logo.

### PR DESCRIPTION
**Pull request by subodhk01**

### CODE REVIEW

**Comment by subodhk01** [comment]
issue #72

**Comment by subodhk01** [comment]
@shivanshs9

File: static/css/dropdown-menu.css [hunk 1]
```diff
@@ -0,0 +1,5 @@
+.dropdown-menu {
+    background: black;
+    border-radius: 10px;
+    box-shadow: 0px 0px 9px 2px rgb(73, 73, 73);
```
Comment (shivanshs9) [inline static/css/dropdown-menu.css]: Can you lighten the shadow a bit? Right now, it looks quite a lot raise in light theme on white background

**Review by shivanshs9 — CHANGES_REQUESTED** [review]

**Review by shivanshs9 — CHANGES_REQUESTED** [review]
If you'll check the Inspect element, you'll notice the dropdown-menu has two CSS styles applied on it, one by your new CSS file and the other by the existing SCSS. Doesn't make sense to apply style on one element by 2 files, especially since that CSS code has **-webkit-box-shadow** with different value and you have **box-shadow** only. In some earlier browsers, the older value will be applied leading to confusion.
Also I suggest you to just use background and border-radius only and let the box-shadow be the same.

**Review by shivanshs9 — APPROVED** [review]
LGTM!

**Pull request merged**